SimCalorimetry/EcalSimProducers/interface/EcalDigiProducer.h

```diff
@@ -15,7 +15,7 @@
 #include "SimCalorimetry/EcalSimAlgos/interface/EcalDigitizerTraits.h"
 #include "SimCalorimetry/EcalSimAlgos/interface/EcalTDigitizer.h"
 #include "SimGeneral/MixingModule/interface/DigiAccumulatorMixMod.h"
-
+#include "SimCalorimetry/EcalSimAlgos/interface/EBHitResponse.h"
```

Comment:
please revert the unnecessary changes in this file

Reply:
not done?

Reply:
still not done

Reply:
Not sure to see what is the problem

Reply:
This PR makes changes to this file that are not necessary for the purpose of the PR. Look at the diff on GitHub, and revert any changes that are not necessary, e.g. replacing forward declaration with header include.